### PR TITLE
closes #12 - This changes the format of the cpf and cnpj claims to string

### LINKED ISSUE #12: ICP Brasil CNPJ Field
We need the Security WG to clarify about the format of the CNPJ. 
On the ICP Profile it says it's a number of 14 digits. 

Considering the issues we may find with the leading zeroes on the CNPJ we need to understand if it must be stored as a string, or as a number of up to 14 digits, zero padded. After this definition we will set it accordingly on the json libraries.

### PR DESCRIPTION
**Pull request by RalphBragg**
closes #12
A note has been added on the documentation tickets to make sure the examples are updated to reflect

### CODE REVIEW

**Comment by RalphBragg** [comment]
Waiting review of the markdown before generating the additional files (HTML and XML) for publication

File: open-banking-brasil-financial-api-1_ID1.md [hunk 1]
```diff
@@ -251,8 +252,8 @@ This profile defines "cnpj" as a new standard claim as per
 
 **CNPJ**, short for Cadastro Nacional de Pessoas Jurídicas, is an identification number
  of **Brazilian** companies issued by the **Brazilian** Ministry of Revenue, **in**
- Portuguese "Secretaria da Receita Federal" or "Ministério da Fazenda". In the Brasil open banking identity model,
- individuals can associated with 0 or more CNPJs. A CNPJ is an number that is either 8 or 14 digits long,
+ Portuguese "Secretaria da Receita Federal" or "Ministério da Fazenda". In the Brasil Open Banking identity model,
+ individuals can associated with 0 or more CNPJs. A CNPJ is a string consisting of numbers that is either 8 or 14 digits long and may start with a 0,
```
Comment (EkkeErick) [inline open-banking-brasil-financial-api-1_ID1.md:256]: Change text to:

**(change starts)**
individuals can be associated with 0 or more CNPJs. A CNPJ is a string consisting of numbers 14 digits long and may start with one or more 0,
**(change ends)**

Just to clarify the 8 digits code is called ISPB code which is a code used by the central bank to define an organization.

It is composed by the 8 first digits of the CNPJ, however it's not exactly called CNPJ

**Review by EkkeErick — CHANGES_REQUESTED** [review]
Just that one change and we are cool

**Review by EkkeErick — APPROVED** [review]
Ok. The Cnpj Looks Good to Me

**Pull request merged**